lib/logux/channel_controller.rb

```diff
@@ -13,5 +13,9 @@ def initial_data
     def initial_meta
       { clients: [meta.client_id] }
     end
+
+    def since_time
+      action['since'].try(:[], 'time')
```

Comment:
We have no automatic type casting here, and it will appear probably as numeric instance here, so we have to covert it to the DateTime using:
````ruby
Time.at(seconds_since_epoch_integer).to_datetime
```

Reply:
Good idea :+1: